toolchain/check/context.h

```diff
@@ -793,6 +793,8 @@ class Context {
   // Per-import constant values. These refer to the main IR and mainly serve as
   // a lookup table for quick access.
   //
+  // Indexed by the integer values of the `SemIR::ImportIRId`s in import_irs().
```

Comment:
```suggestion
  // Per-import constant values, indexed by `SemIR::ImportIRId`. These refer to
  // the main IR and mainly serve as a lookup table for quick access.
```

I'm a little confused by what the integer values in import_irs are; aren't these just indexed by ImportIRId?

Reply:
Yeah, this is maybe more verbose than needed, but represents my thinking as I was learning. `ImportIRId` is an `IdBase` which is not a number, it's a class with an integer `index` value inside it, and it's indexed by that number. Being more familiar with this now, I may also have just written this as "by ImportIRId", so we can adjust that.